_studio/shared/src/mfx_static_assert_structs.cpp

```diff
@@ -350,6 +350,10 @@
         MSDK_STATIC_ASSERT_STRUCT_SIZE(mfxExtDecVideoProcessing     ,132  )
         MSDK_STATIC_ASSERT_STRUCT_SIZE(mfxExtChromaLocInfo          ,32   )
         MSDK_STATIC_ASSERT_STRUCT_SIZE(mfxExtMBQP                   ,64   )
+        MSDK_STATIC_ASSERT_STRUCT_SIZE(mfxQPandMode                 ,4    )
```

Comment:
```MFX_VERSION``` guard is missing

Reply:
I'm going to get rid of them soon.

Reply:
But you have such guards introduced by this commit. I support your intention to remove them, but for now I'm just asking for consistency.